kubernetes/gke-utility/argocd/argocd-sa.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    iam.gke.io/gcp-service-account: argocd@k8s-infra-prow.iam.gserviceaccount.com
+  name: argocd-application-controller
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    iam.gke.io/gcp-service-account: argocd@k8s-infra-prow.iam.gserviceaccount.com
```

Comment:
Are both SAs supposed to have the same annotation?

Reply:
Yes